syntax = "proto2";

message PandaLensError {
  optional string error = 1 [default = ""];
}